BuildAndTest.proj

```diff
@@ -15,9 +13,11 @@
     <RunTestArgs Condition="'$(TestVsi)' == 'true'">$(RunTestArgs) -testVsi</RunTestArgs>
     <RunTestArgs Condition="'$(Trait)' != ''">$(RunTestArgs) -trait:$(Trait)</RunTestArgs>
     <RunTestArgs Condition="'$(NoTrait)' != ''">$(RunTestArgs) -notrait:$(NoTrait)</RunTestArgs>
+    <!-- DO NOT MERGE -->
```

Comment:
Yep.  @agocke and I are cleaning up all the remaining "DO NOT MERGE" entries.  Doing a full grep before we "finish".